docker/wrapper.py

```diff
@@ -94,29 +94,12 @@ def generate_manifest(sample_tars, sample_singles, sample_pairs, workdir, output
 
 
 def catFiles(outputFile, inputFiles):
-    with open(outputFile, 'w') as outfile:
-        for fname in inputFiles:
-            with open(fname) as infile:
-                for line in infile:
-                    outfile.write(line)
-    return outputFile
-
-def gzipCatFiles(outputFile, inputFiles):
-    with gzip.open(outputFile, 'w') as outfile:
-        for fname in inputFiles:
-            with gzip.open(fname) as infile:
-                for line in infile:
-                    outfile.write(line)
-    return outputFile
-
-
-def zcatFiles(outputFile, inputFiles):
     '''
-    Routine to unzip and concatenate input files
-    using zcat which is much faster than using python 
-    gunzip
+    Routine to concatenate input files, gzipped or not using cat
+    For gzipped files this is much faster than using python gunzip
+    see https://www.biostars.org/p/136025/ and https://www.biostars.org/p/81924/
     '''
-    command = 'zcat'
+    command = 'cat'
```

Comment:
Yes I tried on multiple samples. It does actually work for gzipped files. Take a look at  https://www.biostars.org/p/136025/ and https://www.biostars.org/p/81924/